docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
I think you've oversold the "pro" in "pro tip"

Oooo! Burn!

Also, you're all jerks. Is this what it's like when there's not enough code to review? Go fix some bugs! :-D :-D :-D